app/jobs/regular/create_chat_mention_notifications.rb

```diff
@@ -17,46 +17,47 @@ def execute(args = {})
       @user_ids_to_group_mention_map = args[:user_ids_to_group_mention_map] || {}
       @memberships.each do |membership|
         unless DiscourseChat::ChatNotifier.user_has_seen_message?(membership, @chat_message.id)
-          group_name = @user_ids_to_group_mention_map[membership.user.id.to_s]
-          mention_type = group_name.present? ?
-            :chat_group_mention :
```

Comment:
We should add a comment in core to note that the `chat_group_mention` notification type is no longer in use. It probably should stay reserved, for historical reasons.